src/main/java/cz/cvut/kbss/termit/persistence/dao/TermDao.java

```diff
@@ -1,16 +1,13 @@
 /**
  * TermIt Copyright (C) 2019 Czech Technical University in Prague
  * <p>
- * This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public
- * License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later
- * version.
+ * This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by
```

Comment:
What is the point of reformatting all the stuff around?

Reply:
I have changed my default line length in IDEA to 140 characters, so it reformatted the license header. If it is a big problem, I can revert it.